netreg_benchmark.py

```diff
@@ -158,10 +162,11 @@
     os.makedirs(args.networks_dir)
 
 network_filename = os.path.join(args.networks_dir,
-                                'sim_groups_p{}_e{}_u{}_s{}.tsv'.format(
+                                'sim_groups_p{}_u{}_a{}_r{}_s{}.tsv'.format(
                                     args.num_features,
-                                    args.noise_stdev,
```

Comment:
Doesn't matter really, but why are you no longer specifying the noise standard deviation in the file name? It looks like it's still a used parameter in your add/remove model. Just checking that that was intentional.

Reply:
In my current set of experiments I'm not changing the noise standard deviation (it's fixed at 0.1 in `scripts/run_netreg_benchmark.py`) so I got rid of it to cut down on the number of parameters in the filenames.

I'll make a note to make the naming scheme more general in the future.